lib/mnesia/test/ext_test_server.erl

```diff
@@ -390,8 +421,8 @@ handle_call({repair_continuation, Cont, Ms}, _From, State) when element(1, Cont)
     ?DBG({repair_continuation, ext_disc_only_copies, Cont, Ms}),
     Res = ?TRY(dets:repair_continuation(Cont, Ms)),
     {reply, Res, State};
-handle_call({repair_continuation, ext_ram_copies, Cont, Ms}, _From, State) ->
-    ?DBG({repair_continuation, Cont, Ms}),
+handle_call({repair_continuation, Cont, Ms}, _From, State) ->
+    ?DBG({repair_continuation, ext_ram_copies, Cont, Ms}),
```

Comment:
I think this was a bug - was getting a function clause error since a 3-tuple is passed